drivers/storage/portworx/component/csi.go

```diff
@@ -686,6 +699,54 @@ func getCSIDeploymentSpec(
 		)
 	}
 
+	if csiConfig.IncludeHealthMonitorController && healthMonitorControllerImage != "" {
+		deployment.Spec.Template.Spec.Containers = append(
+			deployment.Spec.Template.Spec.Containers,
+			v1.Container{
+				Name:            csiHealthMonitorControllerContainerName,
+				Image:           healthMonitorControllerImage,
+				ImagePullPolicy: imagePullPolicy,
+				Args: []string{
+					"--v=3",
+					"--csi-address=$(ADDRESS)",
+					"--leader-election",
+					"--http-endpoint=:8080",
```

Comment:
Is the endpoint supposed to be `:8080`? Just making sure that it is correct.

Reply:
Yes, that's ok - might expose the port as something else later on.

Reply:
I actually meant to check that the format `:8080` is correct.

Reply:
ah yeah that's correct